### PR TITLE
MBS-8303 remove stdout stacktrace spam

### PR DESCRIPTION
**Pull request by dsvoronin**

### CODE REVIEW

File: subprojects/gradle/utils/src/main/kotlin/com/avito/utils/logging/CILoggingDestination.kt [hunk 1]
```diff
@@ -25,7 +25,6 @@ object StdoutDestination : CILoggingDestination() {
         println(formattedMessage.message)
         formattedMessage.cause?.let { throwable ->
             println(throwable.message)
-            throwable.printStackTrace()
```
Comment (sergioboi) [inline subprojects/gradle/utils/src/main/kotlin/com/avito/utils/logging/CILoggingDestination.kt:28]: Do we lose a stacktrace now?

Comment (dsvoronin) [reply subprojects/gradle/utils/src/main/kotlin/com/avito/utils/logging/CILoggingDestination.kt:28]: it's available in sentry
you can find it via error message

Comment (sergioboi) [reply subprojects/gradle/utils/src/main/kotlin/com/avito/utils/logging/CILoggingDestination.kt:28]: It's hard to find all stacktraces related to build
Could we generate a link to the Sentry with applied filters to see all reported error by current build?

Comment (dsvoronin) [reply subprojects/gradle/utils/src/main/kotlin/com/avito/utils/logging/CILoggingDestination.kt:28]: great idea! will do in separate PR

**Review by int02h — APPROVED** [review]

**Pull request merged**